src/olympia/reviews/views.py

```diff
@@ -432,5 +432,6 @@ def flag(self, request, *args, **kwargs):
         request = request._request
         response = flag(request, addon.slug, kwargs.get('pk'))
         if response.status_code == 200:
+            response.content = ''
```

Comment:
What was that before? I'm missing a test that tests that change. Actually, ugh, `flag` serializes things to json and stuff and we're throwing that away? I didn't notice that before, damn.